include/bluetooth/l2cap.h

```diff
@@ -132,6 +129,9 @@ struct bt_l2cap_le_chan {
 	/** Segment SDU packet from upper layer */
 	struct net_buf			*_sdu;
 	u16_t				_sdu_len;
+
+	struct k_work			rx_work;
```

Comment:
This is part of the cleanup that I filed an issue for. *Most* of this trust can/should be compiled out without DYNAMIC_CHANNEL, but since I wanted to keep this PR minimal (due to the upcoming release I decided to leave this detail alone for now.